vitrivr-engine-core/src/main/kotlin/org/vitrivr/engine/core/model/content/factory/CachedContentFactory.kt

```diff
@@ -119,7 +120,9 @@ class CachedContentFactory : ContentFactoriesFactory {
 
         override fun newMeshContent(model3D: Model3D): Model3DContent {
             check(!this.closed) { "CachedContentFactory has been closed." }
-            TODO()
+            val content = InMemoryMeshContent(model3D) /* TODO: Caching. */
+            logger.warn { "Caching of MeshContent is not yet implemented. Using in-memory content instead." }
+            return content
```

Comment:
I guess we can merge this prior to #97

Reply:
Agree! I wouldn't want to wait for this.